src/components/app/app.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { useRoutes } from 'react-router-dom';
 import routes from 'constants/routes';
 import { Page } from 'pages/page';
+import { AuthProvider } from 'context';
+import { GoogleOAuthProvider } from '@react-oauth/google';
 
 export const App = () => {
   const appRoutes = useRoutes(routes);
 
   return (
-    <>
-      {/* {'react-trainee-project'} */}
-      {appRoutes}
-      <Page />
-    </>
+    <AuthProvider>
+      <GoogleOAuthProvider clientId={process.env.CLIENT_ID || ''}>
+        {appRoutes}
+        <Page />
```

Comment:
maybe `<Page>{appRoutes}</Page>` ?
and better solution will be move Page to routes

Reply:
dont understand what did you mean about move page to routes

Reply:
done